pkg/resources/row_access_policy.go

```diff
@@ -118,101 +178,112 @@ func ReadRowAccessPolicy(d *schema.ResourceData, meta interface{}) error {
 		return nil
 	}
 	if err := d.Set(FullyQualifiedNameAttributeName, id.FullyQualifiedName()); err != nil {
-		return err
+		return diag.FromErr(err)
 	}
 
 	if err := d.Set("name", rowAccessPolicy.Name); err != nil {
```

Comment:
why do we set the required fields (name, schema and db)?

Reply:
Removed and added in the import function.